edubots_surveys.Rnw

```diff
@@ -303,7 +303,7 @@ In the framework of the EduBots KA2 European project\footnote{https://www.edubot
 % JJ - feel free to add one 
 % Antonio - The thing is I don't catch why are we saying that, since it is supossed that we should 'defend' the use of chatbots (from the project perspective, at least). Or if we don't want to defend them, at least not saying that they are useless most of times. XD
 % Moreover, considering that in the following paragraphs we are trying to 'sell' them.
-
+A chatbot is a program, sometimes developed using Artificial Intelligence techniques, that allows a computer to communicate in a similar way as humans do \cite{Gong2008}, using natural language. Indeed, most of them would be, apparently, close to passing a classic Turing test \cite{ moor2003turing }, since they are able to answer almost any question fluently, and even pose questions in their conversations. The recent advancement in this discipline has played an important role in many fields especially in education and online tutoring \cite{clarizia2018,Smutny2020}. These automatic systems facilitate the delivery of personalized learning by adapting to students’ pace of learning and providing customized online tutoring outside the classroom. Thus, chatbots can significantly contribute to providing interactive learning experiences as well as improving individual attention \cite{agarwal2020}.
```

Comment:
Can you please again delete whatever this is intended to substitute? I kind of think it's the paragraph below (mainly because of citations), but I'm not sure...